app/app.go

```diff
@@ -254,11 +258,22 @@ func NewBandApp(
 
 	app.sm.RegisterStoreDecoders()
 
+	lanes := CreateLanes(app)
+
+	// create Band mempool
+	bandMempool := mempool.NewMempool(app.Logger(), lanes)
+	// set the mempool
+	app.SetMempool(bandMempool)
+
 	// Initialize stores.
 	app.MountKVStores(app.GetKVStoreKey())
 	app.MountTransientStores(app.GetTransientStoreKey())
 	app.MountMemoryStores(app.GetMemoryStoreKey())
 
+	feedsMsgServer := feedskeeper.NewMsgServerImpl(app.FeedsKeeper)
+	tssMsgServer := tsskeeper.NewMsgServerImpl(app.TSSKeeper)
+	oracleMsgServer := oraclekeeper.NewMsgServerImpl(app.OracleKeeper)
```

Comment:
should we just send keeper instead of MsgServer and convert in TxMatchHandler?
think it's more flexible when we need something from main keeper.

Reply:
I think we should try to send only what is used